api/api/spec/spec.yaml

```diff
@@ -6084,7 +6084,9 @@ components:
     purge:
       in: query
       name: purge
-      description: "Permanently delete an agent from the key store"
+      description: |
+        Permanently delete an agent from the key store. The wazuh-authd `purge` configuration option has 
```

Comment:
How about linking to https://documentation.wazuh.com/4.13/user-manual/reference/ossec-conf/auth.html#purge ?

If not, my suggestion is  making reference of `wazuh-authd `purge` configuration` as `The `purge` option, part of `auth` configuration block`

Reply:
Adding a documentation link would require us to maintain it on every realese, if the URL or the page content changes, we would have to update the spec. That's why I didn't reference it directly.

Regarding the other suggestion, I didn't understand it. Is it mentioning the specific place where that configuration is?